### ISSUE TITLE
Add support for tolerations in WebsphereLibertyApplication CRD

### ISSUE THREAD

**Issue by claus-klp** [WASdev/websphere-liberty-operator#652]
We need our Liberty pods to run on dedicated nodes in our clusters and crucially keep all other pods away from these nodes. (The Liberty pods are just a small part of our total number of pods in the cluster).

This would have been very easy to solve with taints and tolerations:
-	Set taint on the dedicated nodes for liberty (this we do ourselves)
-	Set toleration for said taint on the liberty deployments/pods (this we expected to be able to do through the `WebsphereLibertyApplication` CRD)

Our "workaround" today has been to add a custom-label on all our non-liberty nodes and then set a node-selector on all our "non-liberty-pods" towards that label.

TLDR; it would be more elegant and less maintenance/error-prone for us if we could set tolerations through the `WebsphereLibertyApplication` CRD. Can this be implemented?

The field within a deployment we would like to manage:
```
spec:
  template:
    spec:
      tolerations: <our-custom-liberty-node-toleration>
```

---

- [x] RCO
- [x] OLO
- [x] WLO
- [x] Tests
- [x] Docs

**Comment by claus-klp** [comment]
We created this issue as agreed :) @leochr

**Comment by leochr** [comment]
Planning to deliver this functionality in the release planned for the end of September / beginning of October

**Comment by leochr** [comment]
@claus-klp WebSphere Liberty Operator 1.4.0 was released yesterday and is available from IBM Operator Catalog. It includes the support for tolerations. Use subscription channel `v1.4` to install/upgrade. Here is the relevant doc: 
https://www.ibm.com/docs/en/was-liberty/base?topic=operator-configuration-examples#cfg-r-commoncomp__cc-tolerations__title__1

**Comment by leochr** [comment]
Completed. Closing
